compose.yaml

```diff
@@ -53,6 +53,25 @@ services:
       db:
         condition: service_healthy
 
+  git-landing-worker:
+    image: lando
+    command: lando start_landing_worker git
+    environment:
+      DEFAULT_GRACE_SECONDS: 10
+      LANDING_WORKER_USERNAME: app
+      # See [0] and [1] to setup up and install a GitHub app
+      # [0] https://docs.github.com/en/enterprise-cloud@latest/apps/creating-github-apps/registering-a-github-app/registering-a-github-app
+      # [1] https://docs.github.com/en/enterprise-cloud@latest/apps/using-github-apps/installing-your-own-github-app
+      # GITHUB_APP_ID: 123456789
+      # GITHUB_APP_PRIVKEY: |
+      #  -----BEGIN RSA PRIVATE KEY-----
+      #  ...
```

Comment:
Should these be uncommented for testing the Github app auth? It's unclear to me what this comment is for.

Reply:
It's meant to be documentation of sort. 
As this requires a dedicated app registration, and a private key, it can't be setup by default in the repo and/or I don't think we should ship a key, albeit a test one.

I left that here as a mind-jogger of how to configure it. Arguably, I should probably also update the readme.

Reply:
I left that here, but added doc about the most important parameters in the README, too.

Reply:
I had a change of heart 😬 I moved everything neatly into the README.